### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DominikDary**
@gauripowale please review and merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,3 +103,15 @@ enough tests pass to qualify the new operating system, then either merge the pul
 the pull request with bug fixes. If they just want to retrigger the tests due to potential test flakiness, they should
 push a new commit to do so, which would include a change to publish_and_test_config.yaml with the field
 'publish_dcos_images_after: never' to avoid rebuilding identical new images.
+
+## Running Locally
+
+This code can be executed locally by following theose steps:
```
Comment (jeschkies) [inline README.md:109]:
```suggestion
This code can be executed locally by following these steps:
```

File: README.md [hunk 2]
```diff
@@ -105,3 +103,15 @@ enough tests pass to qualify the new operating system, then either merge the pul
...
+
+1) Add a your ssh public key to your AWS account
```
Comment (jeschkies) [inline README.md:111]: Why is this required?

File: README.md [hunk 3]
```diff
@@ -105,3 +103,15 @@ enough tests pass to qualify the new operating system, then either merge the pul
 the pull request with bug fixes. If they just want to retrigger the tests due to potential test flakiness, they should
 push a new commit to do so, which would include a change to publish_and_test_config.yaml with the field
 'publish_dcos_images_after: never' to avoid rebuilding identical new images.
+
+## Running Locally
```
Comment (jeschkies) [inline README.md:107]: Run what locally?

File: README.md (hunk 2 above)
Comment (DominikDary) [reply README.md:111]: Otherwise the old terraform installer was not working for me.

File: README.md (hunk 3 above)
Comment (gauripowale) [reply README.md:107]: You can run the entire dcos-images automation locally as well - creating a dcos ami through packer, spinning up a cluster through terraform, running integration tests, accessing the cluster to test functionality, etc.

File: README.md [hunk 4]
```diff
@@ -1,8 +1,6 @@
 # dcos-images
 
-The reference for DC/OS images and how to build them.  
-The goal of this repo is to standardize, across all DC/OS users and developers, the usage and build process of OS cloud
-images to run DC/OS.
+Repository is used to qualify DC/OS releases against CentOS and RHEL operating systems.
```
Comment (gauripowale) [inline README.md:3]: You are also using it to qualify FlatCar Linux for the time being right?

File: README.md (hunk 2 above)
Comment (gauripowale) [reply README.md:111]: When running locally, you can configure terraform to use your own ssh-key rather than the default dcos-images ssh-key that is used in the Jenkins automation.

File: README.md (hunk 3 above)
Comment (DominikDary) [reply README.md:107]: Usually the prs are executed within ci to verify the os, docker and dc/os version is working as expected. 
When developing and fixing the scripts / tests, this explains what needs to be done to run it on the developer machine.
Side note: this can be used e.g. for provisioning a cluster / test it that is based on the different OS variants like centOS, rhel or flatcar Linux.

File: README.md (hunk 4 above)
Comment (DominikDary) [reply README.md:3]: We are in the process of figuring out our minimal os strategy.

File: README.md (hunk 2 above)
Comment (jeschkies) [reply README.md:111]: Shouldn't Terraform add the public key to AWS then?

File: README.md (hunk 3 above)
Comment (jeschkies) [reply README.md:107]: Ah got it. Sorry, I thought this was a PR for the DC/OS repo and was confused.
